src/libaktualizr/uptane/tuf.cc

```diff
@@ -137,10 +137,7 @@ Target::Target(std::string filename, const Json::Value &content) : filename_(std
 
     if (custom.isMember("uri")) {
       std::string custom_uri = custom["uri"].asString();
-      // Ignore this exact URL for backwards compatibility with old defaults that inserted it.
-      if (custom_uri != "https://example.com/") {
-        uri_ = std::move(custom_uri);
-      }
+      uri_ = std::move(custom_uri);
```

Comment:
Unfortunately we probably have to keep this special rule in place for backwards compatibility. I wish we could remove it but it will break things if we do.

Reply:
All tests passed, so I am wondering what are the things it will break ?

Reply:
There are thousands of objects in production/SIT whose Target metadata lists a URL as "example.com". If someone were to use a version of libaktualizr that has my recent fix to actually use that URL field but does not know to special-case and ignore that specific URL, they won't be able to update to those targets any more.

Reply:
thanks for explanation.